gatsby/content/projects/clients/hydrogen.mdx

```diff
@@ -0,0 +1,59 @@
+---
+layout: projectimage
+title: Hydrogen
+slug: hydrogen
+categories:
+    - client
+thumbnail: /docs/projects/images/hydrogen-small.png
+description: Web client focusing on personal messaging, performance, offline usage and wide browser support (including mobile!).
+author: Element
+maturity: Beta
+language: JavaScript
+license: Apache-2.0
+repo: https://github.com/vector-im/hydrogen-web/
+home: https://element.io/
+screenshot: /docs/projects/images/hydrogen-large.png
+room: "#hydrogen:matrix.org"
+platforms:
+    - Linux
+    - Mac
+    - Windows
+    - Android
+    - iOS
+    - Web
+featured: true
+features:
+    Room directory: no
+    Room tag showing: no
+    Room tag editing: no
+    Search joined rooms: yes
+    Room user list: yes
+    Display Room Description: yes
+    Edit Room Description: no
+    Highlights: no
+    Pushrules: no
+    Send read markers: yes
+    Display read markers: no
+    Sending Invites: no
+    Accepting Invites: yes
+    Typing Notification: no
+    E2E: yes
+    Replies: yes
+    Attachment uploading: yes
+    Attachment downloading: yes
+    Send stickers: no
+    Send formatted messages (markdown): no
+    Rich Text Editor for formatted messages: no
+    Display formatted messages: yes
+    Redacting: yes
+    Multiple Matrix Accounts: yes
+    New user registration: no
+    voip: no
+    Reactions: yes
+    Message editing: no
+    Room upgrades: no
+    Localisations: 1
+    SSO Support: yes
+---
+
+For more information, see https://github.com/vector-im/hydrogen-web/blob/master/doc/FAQ.md or join #hydrogen:matrix.org!
```

Comment:
What do you think about making a nicer `[our FAQ](https://github.com/vector-im/hydrogen-web/blob/master/doc/FAQ.md)` link?

How about making the room name a link?